tests/error_handler_test.py

```diff
@@ -104,17 +104,29 @@ def test_log_and_exit(cap_out, mock_store_dir):
 
     printed = cap_out.get()
     log_file = os.path.join(mock_store_dir, 'pre-commit.log')
-    assert printed == (
-        'msg: FatalError: hai\n'
-        'Check the log at {}\n'.format(log_file)
-    )
+    printed_lines = printed.split('\n')
+    assert len(printed_lines) == 6, printed_lines
```

Comment:
with pytest you can usually leave out the `, printed_lines` bit -- and pytest'll generate a cascading assertion message for you 👍